bin/MadGraph5_aMCatNLO/cards/production/13TeV/HToAATo4B/SUSY_TTH_TTToAll_HToAATo4B_M-MASS/SUSY_TTH_TTToAll_HToAATo4B_M-MASS_run_card.dat

```diff
@@ -0,0 +1,270 @@
+ F        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.1880  = scale            ! fixed ren scale
+ 91.1880  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.1880  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ 1        = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1        = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
+ 1        = highestmult      ! for ickkw=2, highest mult group
+ 1        = ktscheme         ! for ickkw=1, 1 Durham kT, 2 Pythia pTE
+ 1        = alpsfact         ! scale factor for QCD emission vx
+ F        = chcluster        ! cluster only according to channel diag
+ F        = pdfwgt           ! for ickkw=1, perform pdf reweighting
+ 5        = asrwgtflavor     ! highest quark flavor for a_s reweight
+ T        = clusinfo         ! include clustering tag in output
+ 3.0      = lhe_version       ! Change the way clustering information pass to shower.        
+#*********************************************************************
+#**********************************************************
+#
+#**********************************************************
+# Automatic ptj and mjj cuts if xqcut > 0
+# (turn off for VBF and single top processes)
+#**********************************************************
+   T  = auto_ptj_mjj  ! Automatic setting of ptj and mjj
+#**********************************************************
+#                                                                    
+#**********************************
+# BW cutoff (M+/-bwcutoff*Gamma)
+#**********************************
+  15  = bwcutoff      ! (M+/-bwcutoff*Gamma)
+#**********************************************************
+# Apply pt/E/eta/dr/mij cuts on decay products or not
+# (note that etmiss/ptll/ptheavy/ht/sorted cuts always apply)
+#**********************************************************
+   T  = cut_decays    ! Cut decay products 
+#*************************************************************
+# Number of helicities to sum per event (0 = all helicities)
+# 0 gives more stable result, but longer run time (needed for
+# long decay chains e.g.).
+# Use >=2 if most helicities contribute, e.g. pure QCD.
+#*************************************************************
+   0  = nhel          ! Number of helicities used per event
+#*******************                                                 
+# Standard Cuts
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Minimum and maximum pt's (for max, -1 means no cut)                *
+#*********************************************************************
+ 20  = ptj       ! minimum pt for the jets 
```

Comment:
What's you analysis level cuts on (b)jets? Is 20 GeV cut safe from smearing effects?

Reply:
Yes, the analysis will have a cut of at least 30 GeV on b-jets.

Reply:
But if you have hadronic top the cuts will be applied to those jets in GEN level as well (w > j j).
Is this also fine?

Reply:
Hi @sihyunjeon , so for associated production where you expect the Higgs boost to come from other tree-level particles (i.e. WH, ZH, and ttH) would you recommend to turn off MLM matching and set ptj to 0 or -1?

Reply:
There is no additional jet, i.e. no MLM and ickkw=0. As you enable cut decays and majetflavor 5, ptj will remove b partons below 20 GeV. If you don't cut at analysis level at say 35 GeV for jets, I would reduce this cut.

Reply:
Hi @agrohsje ,
How low can / should I go?  Should I just set it to 0?  Or could this cause other bad behavior somehow?
Thanks,
Andrew

Reply:
Shouldn't matter which value you pick as long as this is well below  analysis. You can just put 1 and will not impact the result.

Reply:
Set ptj to 0